pages/community/tsc.js

```diff
@@ -1,284 +1,300 @@
-import GenericLayout from "../../components/layout/GenericLayout";
-import TSCMembersList from "../../config/MAINTAINERS.json";
-import {sortBy} from 'lodash';
-import NewsletterSubscribe from "../../components/NewsletterSubscribe";
+import GenericLayout from '../../components/layout/GenericLayout';
```

Comment:
@sagarkori143 please turn off any code formatter you are using as it is difficult to review changes here as each and every line has been changed